WcaOnRails/app/models/competition.rb

```diff
@@ -27,7 +27,7 @@ class Competition < ApplicationRecord
            allow_nil: true,
            with_model_currency: :currency_code
 
-  scope :not_over, -> { where("end_date >= ?", Date.today) }
+  scope :not_over, -> { where("end_date >= ? AND results_posted_at IS NULL", Date.today) }
```

Comment:
> The price is an extra DB query.

That could get expensive if we're doing it in a loop over a bunch of competitions =(

> Are they really synced when end_date.nil??

No, but I don't really have any opinion on what should happen when `end_date` is `NULL`. Making `not_over` be the opposite of `is_probably_over` seems like a reasonable behavior.